pages/common/webpack.md

```diff
@@ -0,0 +1,19 @@
+# webpack
+
+> Bundle your js files and other assets into a single output file.
+
+- Create a single output file from an entry point file:
+
+`webpack {{app.js}} {{bundle.js}}`
+
+- Load css files too from your js file. (This uses the css loader for .css files):
+
+`webpack {{app.js}} {{bundle.js}} --module-bind 'css=css'`
+
+- Pass a config file and show compilation progress along with colors:
+
+`webpack --config {{webpack.config.js}} --progress --colors`
```

Comment:
If I remember correctly, `--colors` might be redundant now. I think it might figure out the color support automatically. You would have to check this from the source, though.